roles/openshift_facts/tasks/main.yml

```diff
@@ -6,5 +6,8 @@
     - ansible_version | version_compare('1.9.0', 'ne')
     - ansible_version | version_compare('1.9.0.1', 'ne')
 
+- name: Ensure python-netaddr is installed
+  yum: pkg=python-netaddr state=installed
```

Comment:
@brenton I'd like to go ahead and merge this in as is knowing that we'll have to update it when we pull in support for atomic. Is that Ok with you? If notI can pull the is_atomic fact to this PR but that is currently sitting on the containers branch.